### ISSUE TITLE
[Style guide] Whether to quote values like user names in texts

### ISSUE THREAD

**Issue by PVince81** [owncloud/core#31980]
In various places in ownCloud, some texts are using double quotes around user display names and in others they aren't.

We should decide what approach to take as a style guide and adjust everywhere.

### Case for double quotes

`The user "Patrick Maier" shared "project ideas" with you`

#### Pros
- Clear delimitation from the surrounding text: 

#### Cons
- Doesn't flow well with the text
- Looks technical

### Case for having no delimiter
`The user Patrick Maier shared project ideas with you`

#### Pros
- Looks user friendly

#### Cons
- For some special names the result might become confusing: `The user "user one" shared "folder" with you` vs `The user user one shared folder with you`

### Open questions
- should the quoting be different based on what kind of value it is ? ex: path name vs user name

@jvillafanez @pmaier1 

@felixheidecke I'd appreciate your UX input on this

**Comment by PVince81** [comment]
Or we need to reconsider our wording style to make it flow better with arbitrary names.

**Comment by ownclouders** [comment]
[GitMate.io](https://gitmate.io) thinks possibly related issues are https://github.com/owncloud/core/issues/12425 (Hide users name from sharing text field.), https://github.com/owncloud/core/issues/27007 (Get to see all user names), https://github.com/owncloud/core/issues/5612 (Requesst: User Full Name), https://github.com/owncloud/core/issues/28339 (Wrong Quote Text in personal settings), and https://github.com/owncloud/core/issues/12998 (User Management).

**Comment by pmaier1** [comment]
> The user "Patrick Maier" shared "project ideas" with you

In general I like to keep things simple. This means keeping text short and use highlighting for user names or files.

For the example I would prefer: **Patrick Maier** shared "project ideas" with you.

Platform-dynamic parts are indicated: User names are bold, files are indicated with double quotes (as they might have a file extension and/or look technical anyway)

**Comment by PVince81** [comment]
Great, that could work.

The exception is in plain text locations like plain text emails.

**Comment by pmaier1** [comment]
> The exception is in plain text locations like plain text emails.

Yes, then we should simply output the user name without highlighting, IMO.

**Comment by DeepDiver1975** [comment]
🤖 Automated issue triage by Claude Code review agent.

This issue was opened before 2020 and has had no activity since then, so we are closing it as outdated. If this is still relevant on a current ownCloud release, please **reopen** (or open a fresh issue) with up-to-date details — thank you!
